package.json

```diff
@@ -12,14 +12,15 @@
     "clean": "rimraf ./build && rimraf ./generated",
     "prebuild": "yarn clean",
     "dev": "concurrently \"tsc --watch\" \"nodemon dist/index.js\"",
-    "start": "node ./build/index.js",
-    "graphql:generate": "graphql-codegen --require dotenv/config --config codegen.yml"
```

Comment:
@kevinrodriguez-io as far as I could tell this graphql:generate was not being used? instead the code gen happened when the server booted up / was hot reloaded

Reply:
When you do actually use it, it's really helpful. Though it comes with its own caveats

Reply:
reverting this change for now.